spec/system/vendor_api/vendor_receives_application_spec.rb

```diff
@@ -101,7 +101,7 @@ def then_it_should_include_the_data_from_the_application_form
           uk_residency_status: nil,
           english_main_language: true,
           other_languages: "I'm great at Galactic Basic so English is a piece of cake",
-          english_language_qualifications: '',
+          english_language_qualifications: nil,
```

Comment:
Not for this PR, but maybe we should update `CandidateHelper.candidate_completes_application_form` for the new flow anyway?